libvast/vast/system/explorer.hpp

```diff
@@ -28,13 +28,24 @@
 namespace vast::system {
 
 struct explorer_state {
+  struct event_limits {
+    int64_t total;
+    int64_t per_result;
```

Comment:
Why are these signed integers?

Reply:
I originally had used `-1` to mean 'unlimited' but now it doesnt make sense any more.

Reply:
I think we can just use `std::numeric_limits<T>::max` as default/unlimited now.

Reply:
Yeah, I think we get "unlimited" for free this way.

Reply:
I don't mind doing it, but worth noting that this will then be inconsistent with the semantics of `export_::max_events` where 0 is unlimited.